controllers/clusterresources/redisuser_controller.go

```diff
@@ -339,6 +368,8 @@ func (r *RedisUserReconciler) handleDeleteUser(
 		return err
 	}
 
+	l.Info("Redis User resource/CR was deleted")
```

Comment:
l.Info("Redis User resource/CR was deleted") -> l.Info("Redis User resource was deleted")

I meant that you can choose between 2 options and use only one of them

Reply:
fixed